pkg/R/sparkRBackend.R

```diff
@@ -0,0 +1,93 @@
+# Methods to call into SparkRBackend. 
+
+
+# Returns TRUE if object is an instance of given class
+isInstanceOf <- function(jobj, className) {
+  stopifnot(class(jobj) == "jobj")
```

Comment:
This function actually might lie on the critical path of some programs. Maybe worthwhile to remove this assert